src/Autocomplete/Autocomplete.tsx

```diff
@@ -53,7 +55,7 @@ export default function Autocomplete<
           variant={variant}
         />
       )}
-      renderOption={multiple ? Option : undefined}
+      renderOption={multiple ? Option : OptionWithTooltip}
```

Comment:
Can I not render a tooltip in multiselect mode? Seems like we've coded ourselves out of this option.

Why can't we update the Option component to support mutiple = true | false, option = string | Option with and without tooltip.

Seems like the [renderOption](https://mui.com/material-ui/api/autocomplete/#Autocomplete-prop-renderOption) function interface gives us all the information we'd need to do this.